inlang/source-code/website/src/pages/editor/@host/@owner/@repository/index.page.tsx

```diff
@@ -71,19 +72,30 @@ function TheActualPage() {
 					</p>
 				}
 			>
-				<Match when={lixErrors().some((e) => e.message.includes("401"))}>
-					<RepositoryDoesNotExistOrNotAuthorizedCard code={401} />
+				<Match when={lixErrors().some((err) => err.message.includes("401"))}>
```

Comment:
Using `repo.errors()` seems fitting. Unsure why you need to define lixErrors in state

Reply:
@samuelstroschein  lets take the errors discussion out of this pr, as it was not touched here.